docs/tutorials/quickstart.md

```diff
@@ -96,3 +96,23 @@ To link your own program against your OTIO build using gcc or clang, add the fol
 
 To use opentime without opentimelineio, link with -lopentime instead, and compile with:
 + `c++ -c source.cpp -I/home/someone/cxx-otio-root/include`
+
+# Debugging Quickstart
+
+### Linux / GDB
+
+From your virtual environment, compile with debug flags:
+
++ `env OTIO_CXX_DEBUG_BUILD=1 python setup.py install`
+
+You can then attach GDB to python and run your program:
+
++ `gdb --args python script_you_want_to_debug.py`
```

Comment:
For fun, you could add this to help the lldb users:
`lldb -- python script_you_want_to_debug.py`

I think all the rest applies the same.